src/content/general/support/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 linkTitle: Support
 title: Giant Swarm support
-description: An explanation of how our support service works.
+description: An explanation of how our support service and various specific processes and workflows.
```

Comment:
The new version needs another verb (or to drop "how"), or simply:
```suggestion
description: An explanation of our support service processes and workflows.
```

Reply:
Hmm. We need something longer :o

Reply:
How much longer? Would saying "various support service processes and workflows" make it?